controllers/component_controller.go

```diff
@@ -195,27 +194,7 @@ func (r *ComponentReconciler) UpdateValuesConfigmap(ctx context.Context, logger
 		versionStr := version.Version // https://golang.org/doc/faq#closures_and_goroutines
 		httpDonwloadURLs := version.URLs
 		g.Go(func() error {
```

Comment:
Should we not wait for the goroutine to complete?  The action completed by goroutine is to push the structure into chan. @bjwswang @Abirdcfly